solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
```

Comment:
I think it's complicated as a standalone file. How do we define dev vs prod products? We're going to have to reinvent a lot of Django.